### ISSUE TITLE
Chatter is not live, and that leads to problems

### ISSUE THREAD

**Issue by Yajo** [odoo/odoo#47496]
Impacted versions: all

Steps to reproduce:

1. Log notes in chatter.

Current behavior:

1. Done out of sync.

Expected behavior:

1. Chatter should be synced live, just like the messaging app.

Video/Screenshot link (optional):

![Peek 12-03-2020 12-23](https://user-images.githubusercontent.com/973709/76521414-b2ef0880-645c-11ea-8abf-9f4dc4151068.gif)

@Tecnativa TT22914

**Comment by pedrobaeza** [comment]
That was done on purpose, as that drains totally performance with people keeping a lot of opened tabs (even on night), and was deactivated after a version having it auto-refresh every a time slot (maybe on v8? I don't remember exactly).

Closing as expected behavior.

**Comment by Yajo** [comment]
Why was this a performance issue? There's always live bus connections in all tabs, just subscribing an additional channel doesn't seem to be so problematic once you have that connection open... :thinking:

**Comment by pedrobaeza** [comment]
As said, this was decided on purpose. The reasons are from my own thinking, but let's see if @tde-banana-odoo can tell us more in-depths.

**Comment by Yajo** [comment]
> The reasons are from my own thinking

Then reopen please.

**Comment by pedrobaeza** [comment]
No, because the decision of deactivating is firm since a lot of versions, and that's not going to change. We are only asking gently about knowing the reasons.

**Comment by Yajo** [comment]
> No, because the decision of deactivating is firm since a lot of versions, and that's not going to change.

Can you prove that?

**Comment by pedrobaeza** [comment]
Hehe, I don't have time for that, so let's reopen and let the issue to stay here for long.

**Comment by tde-banana-odoo** [comment]
@seb-odoo

**Comment by seb-odoo** [comment]
I agree with this issue functionally.
The JS code after the OWL refactoring will be ready to handle the update of the chatter seamlessly.

Then the only problem will indeed be to actually get the update from the server. I don't know exactly about the technical reasons of why it wasn't done before.

There are currently no livebus connection in all tabs, because browsers limit the number of HTTP connection to the same domain that can stay open in parallel. Due to this we use one master tab with the longpolling and it is sharing its info with the other tabs. We will try to use websocket in the future to solve this limitation. That being said, it doesn't look like a real limitation here.

Another issue is that we can currently subscribe only to channels, and the chatter is not a channel. This doesn't look like a real limitation either.

Maybe we just want to limit the number of listen/notify at the db level?

So after all the above mentioned points are resolved maybe we will do it, and if I'm wrong and I learn why we still can't, I'll report back here with the reason.

Not sure what is the policy for keeping issues open or not, but it is not a bug and it will not be fixed in stable.

**Comment by pedrobaeza** [comment]
If you have already taken note as feedback, I think we can close this, but I let others to decide (@Yenthe666 @mart-e)

**Comment by Yenthe666** [comment]
I agree with you @pedrobaeza. Let us close this issue as the Odoo team is aware of it and this is not going to change in a stable version for sure. @seb-odoo it would be great if you can leave a message here whenever something has changed related to this issue/task. Could you also link the internal task number please?
